Kitodo/src/main/java/de/sub/goobi/helper/Helper.java

```diff
@@ -507,65 +500,58 @@ public static AktuelleSchritteForm getCurrentTaskForm() {
         return currentTaskForm;
     }
 
-    public static final FilenameFilter imageNameFilter = new FilenameFilter() {
-        @Override
-        public boolean accept(File dir, String name) {
-            boolean fileOk = false;
-            String prefix = ConfigCore.getParameter("ImagePrefix", "\\d{8}");
-
-            List<String> regexList = new ArrayList<>();
-            regexList.add(prefix + "\\.[Tt][Ii][Ff][Ff]?");
-            regexList.add(prefix + "\\.[jJ][pP][eE]?[gG]");
-            regexList.add(prefix + "\\.[jJ][pP][2]");
-            regexList.add(prefix + "\\.[pP][nN][gG]");
-            regexList.add(prefix + "\\.[gG][iI][fF]");
-
-            for (String regex : regexList) {
-                if (name.matches(regex)) {
-                    fileOk = true;
-                }
-            }
+    public static final FilenameFilter imageNameFilter = (dir, name) -> {
```

Comment:
Missing JavaDoc for public method. Or move this code into its own class to get ride out of this inner class usage.

Reply:
JavaDocs can't be made for lambdas - param attributes are not allowed there - and it is constant.

Reply:
I would move this code to its own classes with proper JavaDoc and explanation and add tests for this class to prove its correct implementaion. Maybe I'm a old school programmer but this is more readable then this lambda expression.

Reply:
I have simplified this two lambdas.